ios/ZulipMobile/Release.xcconfig

```diff
@@ -0,0 +1,8 @@
+//
+//  Release.xcconfig
+//  ZulipMobile
+//
+//  This file is in .gitignore. Add your own key to DEVELOPMENT_TEAM
+
+CODE_SIGN_IDENTITY = iPhone Developer
+DEVELOPMENT_TEAM =
```

Comment:
One could probably put a provisioning script in place that copies from the example file, but I don't think it's likely to be worth it...